BTrees/tests/common.py

```diff
@@ -77,6 +94,38 @@ def _closeRoot(self, root):
         transaction.abort()
         root._p_jar.close()
 
+    def testPersistentSubclass(self):
+        # Can we subclass this and Persistent?
+        # https://github.com/zopefoundation/BTrees/issues/78
+        import persistent
+
+        class PersistentSubclass(persistent.Persistent):
+            pass
+
+        __traceback_info__ = self._getTargetClass(), persistent.Persistent
+        type('Subclass', (self._getTargetClass(), PersistentSubclass), {})
+
+    def testPurePython(self):
+        import importlib
+        kind = self._getTargetClass()
+        class_name = kind.__name__
+        module_name = kind.__module__
+        module = importlib.import_module(module_name)
+
+        # If we're in pure python mode, our target class module
+        # should not have an '_' in it (fix_pickle changes the name
+        # to remove the 'Py')
+
+        # If we're in the C extension mode, our target class
+        # module still doesn't have the _ in it, but we should be able to find
+        # a Py class that's different
+
+        self.assertNotIn('_', module_name)
+        self.assertIs(getattr(module, class_name), kind)
+
+        if not PURE_PYTHON and 'Py' not in type(self).__name__:
+            self.assertIsNot(getattr(module, class_name + 'Py'), kind)
```

Comment:
Trying to understand this test hurts my brain, so instead I'll just trust you. ;)